test/powershell/Language/Scripting/NativeExecution/NativeCommandProcessor.Tests.ps1

```diff
@@ -0,0 +1,24 @@
+Describe "Native Command Processor" -tags "CI" {
+    # Find where test/powershell is so we can find the createchildprocess command relative to it
+    $powershellTestDir = $PSScriptRoot
```

Comment:
This should be put in a BeforeAll. Test code should not exist outside a defined scope for Pester.